modules/util/portal-tools-service-builder/src/main/java/com/liferay/portal/tools/service/builder/ServiceBuilder.java

```diff
@@ -7880,6 +7894,12 @@ private void _write(
 
 	private static final int _MAX_LINE_LENGTH = 80;
 
+	private static final String _NEW_HBM_NAMESPACE_PREFIX =
```

Comment:
@yuhai Could you rename these constants to `_HIBERNATE_5_HBM_NAMESPACE` and `_HIBERNATE_3_HBM_NAMESPACE`? So we know clearly that one is for 5 and one is for 3.

Reply:
@dantewang, yes

Reply:
@yuhai Oh, could you put the full namespace url into the constant? So instead of

```
http://www.hibernate.org/dtd
```

do
 
```
http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd
```

Reply:
@dantewang, ok. I will close the pr and then resend it.